--- libzbc-5.8.5/tools/gzbc/gzbc_if_dev.c.orig	2020-03-09 06:44:41.000000000 +0100
+++ libzbc-5.8.5/tools/gzbc/gzbc_if_dev.c	2020-05-30 20:16:54.056454623 +0200
@@ -1441,7 +1441,7 @@
 						GTK_DIALOG_MODAL | GTK_DIALOG_DESTROY_WITH_PARENT,
 						GTK_MESSAGE_ERROR,
 						GTK_BUTTONS_OK,
-						str);
+						"%s", str);
 		gtk_message_dialog_format_secondary_text(GTK_MESSAGE_DIALOG(dialog),
 							 "Error %d (%s)",
 							 ret, strerror(ret));
